src/main/java/com/zendesk/maxwell/MaxwellFilter.java

```diff
@@ -130,6 +149,24 @@ public boolean matches(String database, String table) {
 		return matchesDatabase(database) && matchesTable(table);
 	}
 
+	private boolean matchesValues(Map<String, Object> data) {
+		for (Map.Entry<String, String> entry : includeColumnValues.entrySet()) {
+			String column = entry.getKey();
+
+			if (data.containsKey(column)) {
+				String columnValue = entry.getValue();
+
+				Object value = data.get(column);
+				if (value == null) return false;
+
+				String valueString = value.toString();
+				if (!columnValue.equals(valueString)) return false;
```

Comment:
In the future; regexp matching on the values would be nice.

Reply:
Hi, did you mean that we should have a separated folk on maxwell, then we add the the fix for Google Pub/Sub, then submit a new pull request for it?

Reply:
I just mean remove the pub-sub fix from this PR and open it as a new PR.